business-central-parent/business-central-home-page-product/src/main/resources/org/kie/bc/client/resources/i18n/Constants_es.properties

```diff
@@ -14,8 +14,8 @@
 # limitations under the License.
 #
 
-Heading=Bienvenido a Red Hat Process Automation Manager
-SubHeading=Red Hat Process Automation Manager (RHPAM) ofrece un conjunto de herramientas flexibles, que apoyan la manera en que usted necesita trabajar. Seleccione una herramienta a continuación para empezar.
+Heading=Bienvenido a IBM Business Automation Manager Open Editions
+SubHeading=IBM Business Automation Manager Open Editions (RHPAM) ofrece un conjunto de herramientas flexibles, que apoyan la manera en que usted necesita trabajar. Seleccione una herramienta a continuación para empezar.
```

Comment:
still `RHPAM`? :thinking:

Reply:
it should be BAMOE
```suggestion
SubHeading=IBM Business Automation Manager Open Editions (BAMOE) ofrece un conjunto de herramientas flexibles, que apoyan la manera en que usted necesita trabajar. Seleccione una herramienta a continuación para empezar.
```

Reply:
just consider RHPAM is in all languages files es, fr, ja and default, so should be replaced on all of them